src/back-end/lib/resources/affiliation.ts

```diff
@@ -182,7 +182,7 @@ const resource: Resource = {
         }
 
         const existingAffiliation = validatedAffiliationId.value;
-        if (existingAffiliation.membershipType === MembershipType.Owner && await readActiveOwnerCount(connection, request.query.organization) === 1 ) {
+        if (existingAffiliation.membershipType === MembershipType.Owner && await readActiveOwnerCount(connection, existingAffiliation.organization.id) === 1 ) {
```

Comment:
@sutherlanda I think you forgot to change this ref from the query string to the `existingAffiliation`. Let me know if the query param ref was intentional.

Reply:
Yes that was an oversight, thank you.